### DISCUSSION TITLE
How to rename online project after creating it?

### DISCUSSION THREAD

**Discussion by danieltomasz** [typst/typst#2694] [Q&A] [answered]
How to rename online project after creating it? I don't see this option on https://typst.app/

**Comment by jamesrswift** [comment, marked as answer]
![image](https://github.com/typst/typst/assets/6299280/df04685c-c034-4d13-a36e-61105681fe9a)

**Reply by danieltomasz** [reply]
Thank you!

**Reply by Dherse** [reply]
A fellow UGent-er 🎉
